tests/test_analyzer.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from pymc_workflow_analyzer.analyzer import static_analyzer
+
+def test_analyze_script():
+    script_path = 'sample_model.py'
+    expected_repost = {
+        "number_of_import_statements": 6,
+        "imports": ["arviz", "matplotlib.pyplot", "numpy", "pandas", "pymc", "xarray"],
+        "model": [{"name": "Model", "args": [], "kwargs": []}],
+        "distributions": [
+                {"name": "HalfCauchy", "args": ["sigma"], "kwargs": ["beta"]},
+                {"name": "Normal", "args": ["Intercept", 0], "kwargs": ["sigma"]},
+                {"name": "Normal", "args": ["slope", 0], "kwargs": ["sigma"]},
+                {"name": "Normal", "args": ["y"], "kwargs": ["mu", "sigma", "observed"]}
+        ],
+        "samplers": [{"name": "sample", "args": [3000], "kwargs": []}],
+        "math": [],
+        "arviz": [{"name": "plot_trace", "args": ["idata"], "kwargs": ["figsize"]}]
+    }
```

Comment:
Hi @OriolAbril does this look good now?

Reply:
> it could also make sense to store a single integer as "args", indicating how many are used

Oo I missed this point. Here we have to store only the count of args, right?